src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -820,17 +820,21 @@ public static bool TryGetSingleRange(string set, out char lowInclusive, out char
         /// <param name="chars">The span into which the chars should be stored.</param>
         /// <returns>
         /// The number of stored chars.  If they won't all fit, 0 is returned.
+        /// If 0 is returned, no assumptions can be made about the characters.
```

Comment:
Nit, if 0 means "cannot compute" rather than "none" I wonder whether -1 would be clearer.

Reply:
> I wonder whether -1 would be clearer

I don't think so.  If it returns 0, there's no action we can take.  Returning -1 in addition just means we need to do more work for no benefit, on both the producer and the consumer.  You've suggested this a couple times now, so what am I missing?

Reply:
OK fine